db/migrations/mysql/000051_create_msg_root_count.up.sql

```diff
@@ -14,8 +14,8 @@ SET @tmp_count_root := 0;
 	IF(MentionCountRoot_EXIST = 1) THEN
 		SELECT @tmp_count_root := COUNT(*)
 		FROM ChannelMembers
-			WHERE MsgCountRoot = NULL
-			OR MentionCountRoot = NULL;
+			WHERE MsgCountRoot IS NULL
```

Comment:
Is there a difference between the two?

Reply:
Yeah, it seems `NULL` values are not comparable with `=, <, or <>`. Also updated some other occurrences in [79e6b57](https://github.com/mattermost/mattermost-server/pull/19359/commits/70e6b5741af9fd39de45c965c682c4d16e2c3071).